developer-docs-site/sidebars.js

```diff
@@ -437,6 +437,39 @@ const sidebars = {
       className: "sidebar-title",
     },
     "nodes/aptos-api-spec",
+    {
+      type: "category",
+      label: "Aptos SDKs",
+      collapsible: true,
+      collapsed: true,
+      link: { type: "doc", id: "sdks/index" },
+      items: [
+        {
+          type: "category",
+          label: "TypeScript SDK",
+          link: { type: "doc", id: "sdks/ts-sdk/index" },
+          collapsible: true,
+          collapsed: true,
+          items: [
+            "sdks/ts-sdk/typescript-sdk-overview",
+            {
+              type: "category",
+              label: "API client layer",
+              link: { type: "doc", id: "sdks/ts-sdk/sdk-client-layer" },
+              collapsible: false,
+              collapsed: false,
```

Comment:
set these to true

also we're duplicating this section, probably cause this PR started with the older doc site

https://github.com/aptos-labs/aptos-core/blob/main/developer-docs-site/sidebars.js#L251

Reply:
setting these to true, doesnt really hide them - but doesnt show them by default. when clicking the parent page, "api client layer", these pages show up. I was hoping to completely hide them from the sidebar, and you can only get to this page when you click on a link that redirects to this page.

We dont duplicating this section, but separating it into different sections and include more explanations.

Reply:
Docusaurus has really limited JS fun stuff...

Look up at line 246 in this file: https://github.com/aptos-labs/aptos-core/pull/7796/files#diff-4b555969d09d69a2916f46d469f46f0e08925b69f39595e3e2c5913c4a7aa00aL246

The core sdk content is already defined in the sidebar, this is a second version of that.